### DISCUSSION TITLE
What are the settings to improve model detail? (My model's details look terrible).

### DISCUSSION THREAD

**Discussion by denizari82** [alicevision/Meshroom#2284] [Q&A]
I take model photos on a cloudy day and with a mobile phone (Galaxy A23). I photograph from three different angles and by rotating around the model. Here is my example photo and the model result produced: 
![Ekran görüntüsü 2023-12-24 232330](https://github.com/alicevision/Meshroom/assets/93490501/77a39664-cd6b-421f-9a1c-c85a6fea6625)

And of course it produces a very bad result in Blender as well: 
![Ekran görüntüsü 2023-12-24 232932](https://github.com/alicevision/Meshroom/assets/93490501/759ec581-cc6d-4510-9410-aa56fa04b6c7)

By the way, my video card does not have CUDA support and I create a model by deleting all three Nodes as stated here; https://github.com/alicevision/meshroom/wiki/Draft-Meshing

**Where do you think the problem is? And what settings do I make to get better results? Thanks very much for your help.**

**Comment by MrClock8163** [comment]
@denizari82 
Only 3 images for an object like that is way too little. I suggest you take more photos, and instead of rotating the model, leave the model in place, and move yourself. Structure from motion relies on a still environment, by rotating the model you are changing the environment for each photo (shadows will be different, reflections will be different).
With that being said, as the guide you linked explains, without a CUDA enabled GPU, you can only run sparse reconstruction, which will always give lower density results than a dense reconstruction. For a better sparse reconstruction, you'll need more photos.
